MODULE.bazel

```diff
@@ -0,0 +1,32 @@
+module(name = "nanopb")
+
+bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_python", version = "0.31.0")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
+bazel_dep(name = "protobuf", version = "23.1", repo_name = "com_google_protobuf")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    configure_coverage_tool = True,
+    python_version = "3.10",
+    ignore_root_user_error = True,
+)
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+use_repo(pip, "nanopb_pypi")
+pip.parse(
+    hub_name = "nanopb_pypi",
+    requirements_lock = "@nanopb//:extra/requirements_lock.txt",
+    python_version = "3.10",
+)
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
```

Comment:
Brief warning: I don't think Bazel will treat `http_archive` repositories as bzlmod deps. I suspect in this case it's okay because rules_proto_grpc has a working WORKSPACE file, but I can't remember off the top of my head. Just something to keep an eye out for.

Reply:
you're right, it is not considered a bazel_dep. I found one release from bcr, switched to use it

Reply:
Looks like this is still an http archive, did you intend to replace this?